### ISSUE TITLE
Question: What will be the scope of View Model object if it is intialized directly without using ViewModelProvider inside Adapter class ?

### ISSUE THREAD

**Issue by vicgithb** [googlesamples/android-sunflower#240]
PlantAndGardenPlantingsViewModel object is initialized directly inside GardenPlantingAdapter. How will observable fields of this ViewModel behave?

**Comment by XinyueZ** [comment]
would you mind address some details, I cannot catch.

**Comment by XinyueZ** [comment]
Yes, actually you are right, even the google hasn't a solution for ViewModel of Item which should be created from a factory.
However, I think the ViewModel of Item will be considered as a normal object of data-binding.
The idea of MVVM is that everything of business could and should be done in the ViewModel including the Items, never expose logical to UI level.

**Comment by vicgithb** [comment]
Thanks for the reply. I have a quick question. Sunflower is recommended as one of the best starter guide for learning android architecture components. As we know it is in development phase, where can we get the roadmap of upcoming features? Very few google github repos in kotlin are available for Android Jetpack components that address only a specific area only.  

Sunflower is eagerly awaited app, which provides end to end solution. As of now, Sunflower is not using all Architecture component and even for the existing one's full power of Architecture components is not demonstrated as compared to other google codelabs. It's humble request to include more screens and features and if possible share roadmap for upcoming features.

**Comment by Zhuinden** [comment]
...and to provide a working example of master-detail flow using either ConstraintSets, or Navigation AAC :smile:
